Nodejs/Product/Nodejs/Project/NodejsProjectNode.cs

```diff
@@ -145,10 +147,19 @@ public NodejsProjectNode(NodejsProjectPackage package)
         }
 
         private void TryToAcquireTypings(IEnumerable<string> packages) {
+            bool isNewTypingsFolder = !Directory.Exists(Path.Combine(this.ProjectHome, "typings"));
```

Comment:
See reasoning above. This is by-design. In the majority of cases:
a) users will not have an existing typings folder and will only care that a new folder has been added to the project
b) users that have a typings folder in a javascript project will be okay with stuff being downloaded to it

And even the small subset of users that already have a typings folder, but are not okay with stuff being downloaded to it will be educated enough to discover the option.

As mentioned earlier, a suitable next step here is to add an option to use tsd or typings, but that can come in a future PR.